### ISSUE TITLE
how the learning rate change?

### ISSUE THREAD

**Issue by JoyHuYY1412** [facebookresearch/maskrcnn-benchmark#562]

**Comment by chengyangfu** [comment]
Hi, 
   There are two stages of learning rate changes. 
    The first stage is the warmup stage. You can find the reason and explanation in the famous paper 
[Accurate, Large Minibatch SGD: Training ImageNet in 1 Hour](https://arxiv.org/pdf/1706.02677.pdf). In the default setting, `SOLVER.WARMUP_ITERS` is set as 500. It means your first 500 iterations are in the warmup stage and that's the reason why you see the learning rate is much lower the `SOVER.BASR_LR`. 
<br>
      After the first `SOLVER.WARMUP_ITERS` (500) iterations, the learning rate will drop according to the `SOLVER.STEPS` and `SOLVER.GAMMA`(0.1).  The setting of this part is as same as the Detectron and the paper.

**Comment by fmassa** [comment]
Closing following @chengyangfu comment. Thanks @chengyangfu !
